samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/queue/delivery/TimedAssessmentRunnable.java

```diff
@@ -171,7 +196,13 @@ public void run(){
                true,
                SamigoConstants.NOTI_EVENT_ASSESSMENT_TIMED_SUBMITTED));
 
-            notifyGradebookByScoringType(ag, timedAG.getPublishedAssessment());
+            try {
+               notifyGradebookByScoringType(ag, timedAG.getPublishedAssessment());
+            } catch (GradebookServiceException e) {
+               e.printStackTrace();
```

Comment:
Should log instead of printing a stack

Reply:
@yuanhuaqu can you fix?

Reply:
This is all redundant, that is all inside one big try/catch, on purpose, and we want the failure to log with a failure message, if this exception is eaten here, the assessment is still not going to be submitted, it's just going to continue and log a message that it was successful.